src/Chapter4.hs

```diff
@@ -628,8 +666,13 @@ Can you implement a monad version of AND, polymorphic over any monad?
 
 🕯 HINT: Use "(>>=)", "pure" and anonymous function
 -}
+
+-- practice task to better understand binding
+notM :: (Monad m) => m Bool -> m Bool
+notM x = x >>= (\a -> pure (not a))
```

Comment:
Well done! This is indeed a very good practice :+1: That's a very useful task to do to understand monads better :top: 
However, you can notice, that in reality `notM` can be implemented as `fmap not` :slightly_smiling_face: